Modelica/Utilities/System.mo

```diff
@@ -133,12 +128,11 @@ All returned values are of type Integer and have the following meaning:
 </html>"));
   end getTime;
 
-  function getPid "Retrieve the current process id"
+  impure function getPid "Retrieve the current process id"
```

Comment:
This is pure if it runs only during simulation time. Could use a comment that it is intended to stop the value from being calculated at compilation time.

Reply:
I guess you are right here since the PID does not change during simulation.